app/helpers/review_mapping_helper.rb

```diff
@@ -19,6 +19,95 @@ def create_report_table_header(headers = {})
   end
 
   #
+  # This constructs the query to be sent to the sentiment generation server
+  #
+  def construct_sentiment_query(id, review_text)
```

Comment:
If code specific to each metric is added to review_mapping_helper.rb, review_mapping_helper.rb will become huge.  Better to write generic code for querying and parameterize it with the metric that is being used.